speaker/main.go

```diff
@@ -570,13 +571,17 @@ func (c *controller) SetNode(l log.Logger, node *v1.Node) controllers.SyncState
 		}
 	}
 
-	if conditionChanged || labelNodeExcludeBalancersChanged {
+	if conditionChanged || labelNodeExcludeBalancersChanged || schedulableConditionChanged {
 		return controllers.SyncStateReprocessAll
 	}
 
 	return controllers.SyncStateSuccess
 }
 
+func isNodeSchedulableChanged(nodeName string, oldNodes map[string]*v1.Node, newNode *v1.Node) bool {
```

Comment:
let's have a `isNodeAvailableChanged`  function here, so it's going to be easier in case a new condition is added.

Reply:
done